src/EditorFeatures/Core/Implementation/IntelliSense/AsyncCompletion/CommitManager.cs

```diff
@@ -205,6 +205,12 @@ internal CommitManager(ITextView textView, RecentItemsManager recentItemsManager
                 return AsyncCompletionData.CommitBehavior.None;
             }
 
+            if (GetCompletionProvider(completionService, roslynItem) is ICustomCommitCompletionProvider provider)
```

Comment:
❔ Where was this code prior to this change?

Reply:
Snippets still use ICustomCommitCompletionProvider.
We missed snippets in the modern completion. All snippet tests used the legacy completion. When I started moving tests to the modern completion, I found this.

So, for now we have to copy ICustomCommitCompletionProvider related code to the modern completion. This code exists in the legacy one. It is just a copy.


---
In reply to: [325258740](https://github.com/dotnet/roslyn/pull/38569#discussion_r325258740) [](ancestors = 325258740)

Reply:
Was there an issue for this that we can close?

Reply:
No existing issue. Found with this PR.

---
In reply to: [325723181](https://github.com/dotnet/roslyn/pull/38569#discussion_r325723181) [](ancestors = 325723181)